packages/back-end/src/routers/organizations/organizations.controller.ts

```diff
@@ -994,14 +1004,34 @@ export async function signup(req: AuthRequest<SignupBody>, res: Response) {
     }
   }
 
+  let verifiedDomain = "";
+  if (IS_CLOUD) {
+    // if the owner is verified, try to infer a verified domain
+    if (req.userId) {
+      const user = await findUserById(req.userId);
+      if (user?.verified) {
```

Comment:
Same here, we already have `req.verified` and `req.email`, so don't need to fetch user from db